src/map/status.hpp

```diff
@@ -2950,13 +2950,14 @@ struct s_status_change_db {
 	std::vector<sc_type> end;		///< List of SC that will be ended when this SC is activated
 	std::vector<sc_type> fail;		///< List of SC that causing this SC cannot be activated
 	std::vector<sc_type> endreturn;	///< List of SC that will be ended when this SC is activated and then immediately return
+	std::vector<sc_type> cancel;	///< List of SC that will be canceled when this SC ends
 	t_tick min_duration;			///< Minimum duration effect (after all status reduction)
 	uint16 min_rate;				///< Minimum rate to be applied (after all status reduction)
 };
 
 class StatusDatabase : public TypesafeCachedYamlDatabase<uint16, s_status_change_db> {
 public:
-	StatusDatabase() : TypesafeCachedYamlDatabase("STATUS_DB", 2) {
+	StatusDatabase() : TypesafeCachedYamlDatabase("STATUS_DB", 3) {
```

Comment:
```suggestion
	StatusDatabase() : TypesafeCachedYamlDatabase("STATUS_DB", 3, 2) {
```
It is not really a breaking change, right?

Reply:
It's not breaking for it to be missing but I was 50/50 on this since this could break current skill behavior if it's missing, but maybe it's better to make it like you suggested.